Tests/APITesters/AllAPITests/SwiftAPITester/OfferingAPI.swift

```diff
@@ -49,25 +49,29 @@ private func checkCreateOfferingAPI(package: Package) {
     _ = Offering(
         identifier: "",
         serverDescription: "",
-        availablePackages: [package]
+        availablePackages: [package],
+        webCheckoutUrl: URL(string: "https://revenuecat.com")
```

Comment:
This was a breaking change, the fact that these tests failed should have been a warning /cc @aboedo